chunky/src/java/se/llbit/chunky/ui/render/tabs/CameraTab.java

```diff
@@ -221,8 +221,11 @@ private void updateShift() {
       }
     };
     posX.addEventFilter(KeyEvent.KEY_PRESSED, positionHandler);
+    posX.setTooltip(new Tooltip("Camera X coordinate."));
```

Comment:
```suggestion
    posX.setTooltip(new Tooltip("Camera X (west/east) coordinate."));
```

Reply:
That is good extra information; I shall add it, as well as to the other camera coordinate controls.